metacat-common-server/src/main/java/com/netflix/metacat/common/server/properties/AuthorizationServiceProperties.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.common.server.properties;
+
+import lombok.Data;
+
+/**
+ * Properties related to authorization.
+ *
+ * @author zhenl
+ * @since 1.2.0
+ */
+@Data
+public class AuthorizationServiceProperties {
+    private String createAcl;
+    private String deleteAcl;
```

Comment:
We should also add acl for update even though we will not be enabling it initially.